tests/Core/Tokenizer/EnumCaseTest.php

```diff
@@ -0,0 +1,142 @@
+<?php
+/**
+ * Tests converting enum "case" to T_ENUM_CASE.
+ *
+ * @author    Jaroslav Hanslík <kukulich@kukulich.cz>
+ * @copyright 2021 Squiz Pty Ltd (ABN 77 084 670 600)
+ * @license   https://github.com/squizlabs/PHP_CodeSniffer/blob/master/licence.txt BSD Licence
+ */
+
+namespace PHP_CodeSniffer\Tests\Core\Tokenizer;
+
+use PHP_CodeSniffer\Tests\Core\AbstractMethodUnitTest;
+
+class EnumCaseTest extends AbstractMethodUnitTest
+{
+
+
+    /**
+     * Test that the enum "case" is converted to T_ENUM_CASE.
+     *
+     * @param string $testMarker  The comment which prefaces the target token in the test file.
+     * @param string $testContent The token content to expect.
+     *
+     * @dataProvider dataEnumCases
+     * @covers       PHP_CodeSniffer\Tokenizers\PHP::processAdditional
+     *
+     * @return void
+     */
+    public function testEnumCases($testMarker, $testContent)
+    {
+        $tokens = self::$phpcsFile->getTokens();
+
+        $enum = $this->getTargetToken($testMarker, [T_ENUM_CASE, T_CASE], $testContent);
```

Comment:
`$testContent` isn't needed for this one as for both `T_ENUM_CASE` as well as `T_CASE` would have the same content.

It's mostly needed when one of the tokens being searched for is `T_STRING` and there are multiple of the same tokens in the statement after the test marker.

Then again, maybe `T_STRING` should be added to the token array ?

Reply:
> $testContent isn't needed for this one as for both T_ENUM_CASE as well as T_CASE would have the same content.

You're right. Removed.